src/collective/solr/testing.py

```diff
@@ -194,11 +198,13 @@ def activateAndReindex(portal):
 COLLECTIVE_SOLR_FIXTURE = CollectiveSolrLayer()
 
 COLLECTIVE_SOLR_INTEGRATION_TESTING = IntegrationTesting(
-    bases=(LEGACY_COLLECTIVE_SOLR_FIXTURE,),
+    bases=(PLONE_APP_REGISTRY_FIXTURE,
```

Comment:
@tisto With solr config in the registry we need a functional registry for tests. How it get's there I don't care.